### PR TITLE
fix #11557: Pass optimisation arguments to celery_command

### LINKED ISSUE #11557: Can't pass --without-mingle and --without-gossip to celery worker
**Apache Airflow version**: ANY

**Kubernetes version (if you are using kubernetes)** (use `kubectl version`):

**Environment**:

- **Cloud provider or hardware configuration**:
- **OS** (e.g. from /etc/os-release):
- **Kernel** (e.g. `uname -a`):
- **Install tools**:
- **Others**:

**What happened**:

I've an issue with airflow celery worker performance, and I think there are a place for optimization if airflow enables providing --without-gossip while creating celery worker https://github.com/apache/airflow/blob/87038ae42aff3ff81cba1111e418a1f411a0c7b1/airflow/bin/cli.py#L1271 
https://docs.celeryproject.org/en/4.4.3/reference/celery.bin.worker.html?highlight=--without-gossip#cmdoption-celery-worker-without-gossip. In my case, I've a 200 vm and every vm running 2 queues, so 400 **workers exchanging useless message every second** which result in airflow worker is taking about 10% of cpu time (in 4 cores) and to compare, I've a standalone celery app that consume 0.1% of cpu time **after enabling --without-gossip --without-mingle --without-heartbeat**.

**What you expected to happen**:

celery worker should not take more than 2% of cpu time.

**How to reproduce it**:
Using 200 vm and every vm 2 celery workers running at same time, you will see that the celery worker process is taking ~10-15% of cpu time.

PLEASE NOTE:
Due to https://github.com/celery/celery/issues/2566, we can't provide --without-mingle and --without-gossip through configuration (config file).

**Anything else we need to know**:

### PR DESCRIPTION
**Pull request by DreamPearl**
Arguments: --without_mingle and --without_gossip
closes: #11557
related: #11557

---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).

### CODE REVIEW

**Comment by mik-laj** [comment]
CC: @auvipy

File: airflow/cli/commands/celery_command.py [hunk 1]
```diff
@@ -139,6 +139,8 @@ def worker(args):
         'hostname': args.celery_hostname,
         'loglevel': conf.get('logging', 'LOGGING_LEVEL'),
         'pidfile': pid_file_path,
+        'without_mingle': True,
+        'without_gossip': True,
```
Comment (turbaszek) [inline airflow/cli/commands/celery_command.py]: Should it not be configurable by users?

Comment (DreamPearl) [reply airflow/cli/commands/celery_command.py]: Yes, it should. I will fix it shortly.

**Review by turbaszek — APPROVED** [review]

**Review by auvipy — APPROVED** [review]

**Comment by DreamPearl** [comment]
@mik-laj

**Review by mik-laj — APPROVED** [review]

**Review by ashb — APPROVED** [review]

**Pull request merged**